src/data/gfx.yaml

```diff
@@ -1,4 +1,6 @@
 name: Graphics
 summary: Get involved with the [Graphics](https://wiki.mozilla.org/Platform/GFX/Contribute) team
 products: 
- - Core: ['Graphics', 'GFX: Color Management', 'Canvas: WebGL', 'Canvas: 2D', 'ImageLib', 'Graphics', 'Graphics: Layers', 'Graphics: Text', 'Panning and Zooming']
+ - Core: ['Graphics', 'GFX: Color Management', 'Canvas: WebGL', 'Canvas: 2D', 'ImageLib', 'Graphics', 'Graphics: Layers', 'Graphics: Text', 'Graphics: WebRender', 'Panning and Zooming']
+repositories:
+ - servo/webrender: help-wanted, difficulty: easy
```

Comment:
I think it should be `['help-wanted', 'difficulty: easy']`  to be parsed as different string

Reply:
Oops, PR updated again to fix that